.vscode/settings.json

```diff
@@ -33,4 +29,4 @@
   "eslint.format.enable": true,
   "editorconfig.generateAuto": false,
   "files.trimTrailingWhitespace": true
-}
\ No newline at end of file
+}
```

Comment:
Did something format this automatically? 
I [recently removed](https://github.com/ethereum-optimism/optimism/commit/0249d87936317d2b91f649bf318e08779bb92d1b) the new line at EOF. 

It'd be helpful to me if we can not make this change.

Reply:
This was done automatically, can revert, seems like different editors have different preferences

Reply:
Appreciate it, I've struggled to find the setting to make it not happen.